impeller/renderer/blit_pass.cc

```diff
@@ -64,12 +64,12 @@ bool BlitPass::AddCopy(std::shared_ptr<Texture> source,
     return true;  // Nothing to blit.
   }
 
-  // Clip the destination image.
-  source_region = source_region->Intersection(
-      IRect(-destination_origin, destination->GetSize()));
-  if (!source_region.has_value()) {
-    return true;  // Nothing to blit.
-  }
+  // // Clip the destination image.
```

Comment:
This didn't work as expected if the source region isn't the entire size of the source texture. Since I changed the impeller impl to use a translate instead of just a clip we don't hit this case anymore.